package.json

```
@@ -45,8 +45,8 @@
     "@slack/logger": "^4.0.0",
     "@slack/oauth": "^3.0.4",
     "@slack/socket-mode": "^2.0.5",
-    "@slack/types": "^2.16.0",
-    "@slack/web-api": "^7.10.0",
+    "@slack/types": "2.16.0-aiAppsBeta.3",
+    "@slack/web-api": "7.10.0-aiAppsBeta.5",
```

suggestion: I think we'll want to use the newly published packages instead of the beta package:
```suggestion
    "@slack/types": "^2.17.0",
    "@slack/web-api": "^7.11.0",
```

---

@mwbrooks 🙏 ✨